app/utils/database.py

```diff
@@ -970,7 +970,7 @@ def can_edit(self, identifier):
         `identifier` will be parsed as:
 
         `^([a-zA-Z][0-9]+-[0-9]+)$`: individual
-        `^([a-zA-Z][0-9]+)$`: herd
+        `^(([G|M]X1)|[a-zA-Z][0-9]+)$`: herd
```

Comment:
The regular expression `[G|M]` matches one of the three characters `G`, `|`, or `M`.  You want to use `[GM]`, or possibly just `.` (dot) to match any single character before `X1`.

Reply:
Aha I am mixing Javascript good catch, Yes should be GM instead